package.json

```diff
@@ -5,7 +5,12 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "update-openapi.json": "curl https://api.permit.io/v2/openapi.json -o openapi.json",
```

Comment:
Will it work with the `.json` in the `update-openapi.json` ?

Reply:
not sure I understand